README.md

```diff
@@ -272,7 +277,18 @@ e.g., all instance names in the `prdAWS` cluster will begin with `prdAWS-`
 * DB nodes will be named `[prefix]-db`
 * NFS nodes will be named `[prefix]-nfs`
 
-These assumptions hold only for the standard ec2
-Matterhorn clusters; opsworks clusters will use a different
-naming scheme and controlling them will be implemented in a 
-future release.
+#### opsworks clusters
+
+Opsworks cluster naming/tagging assumptions are slightly different.
+In addition, since the ec2-manager should not be used for doing a 
+full start/stop for opsworks clusters, "support" nodes (db, storage)
+are not manipulated.
+
+* The instance name value will be stored in the tag 'Name'
+* Instances are associated with a particular cluster via the
+`opsworks:stack` tag. e.g., all instances in the `foobar` 
+ cluster will be tagged `opsworks:stack=foobar`
+* A cluster will have one admin instance named with the tag `opsworks:layer:admin`
+* Worker nodes will have the tag `opsworks:layer:workers`
+* Engage nodes will have the tag `opsworks:layer:engage`
```

Comment:
Shouldn't matter, but fyi per Greg Logan, it is possible for a MH cluster to have > 1 engage nodes.